### ISSUE TITLE
Error returning from sleep mode

### ISSUE THREAD

**Issue by alystair** [FelixdelasPozas/TrayWeather#58]
Application does not pull weather data successfully after resuming from a long sleep, works A-OK after refreshing by hand. Potentially delay query after waking up?

**Comment by FelixdelasPozas** [comment]
Hello @alystair.

The problem probably comes from disabling the Qt roaming polling of the network (a few versions ago to avoid network performance degradation) so when the computer wakes up doesn't know that the network has restarted also. The network polling is disabled when the "roaming mode" is disabled.
I've made a fix for the next release. I'll update the issue when it is released and leave it open until verifying that it has been solved.

Regards,
Félix.
